docs/developer-guide/maps-configuration.md

```diff
@@ -803,14 +953,14 @@ These are the available options for makers. These are specific of annotations fo
 - `iconColor`: e.g. "red"
 - `iconAnchor`: [0.5,0.5]
 
-##### Multiple rules and filtering
+##### Multiple rules and filtering (deprecated)
 
 In order to support start point and end point symbols, you could find in the style these entries:
 
 - `geometry`: "endPoint"|"startPoint", identify how to get the geometry from
 - `filtering`: if true, the geometry filter is applied.
 
-#### Example
+#### Example (deprecated)
```

Comment:
why this is deprecated? we will be able to have features with multiple style definition or not?